### ISSUE TITLE
Some method score change on the same photo when inference

### ISSUE THREAD

**Issue by zhangxixi0904** [chaofengc/IQA-PyTorch#122]
I download the pretrained model, and run inference_iqa.py to predict the score of samples in ResultsCalibra/dist_dir. And I found some models like hyperiqa will predict different scores on the same photo everytime. 

Although the The gap is not significant, it seems unnormal. I notice the model have already set eval mode. Is there anything that will affect the inference result.

Besides, maniqa and tres have the same problem.

**Comment by chaofengc** [comment]
This is because these models include random cropping and average during testing. I have set the random seed to make sure the results are same in the latest commit [33a2829](https://github.com/chaofengc/IQA-PyTorch/commit/33a28297236364a8db2916c1f64e97c545b6de24).
